### PR TITLE
Add in script to build documentation.  Also changed docker image to be compatible with new CONTENT_DIR structure

### PR DESCRIPTION
**Pull request by cs2be**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ virtualenv venv
 ```
 export ENV="development"
 export SECRET_KEY="my_secret_key"
-export EXTERNAL_TEMPLATE_DIR="<dir where external templates reside>"
+export CONTENT_DIR="<dir where content dir reside>"
```
Comment (varunarora) [inline INSTALL.md]: Do all the references in the app to EXTERNAL_TEMPLATE_DIR need to be updated?

File: portal/requirements.txt [hunk 2]
```diff
@@ -6,3 +6,32 @@ beautifulsoup4==4.6.0
...
+rarfile==3.0
+recommonmark==0.4.0
+scipy==0.19.1
+six==1.11.0
+snowballstemmer==1.2.1
+Sphinx==1.5.6
+sphinx-rtd-theme==0.2.4
+sphinxcontrib-websupport==1.0.1
+typing==3.6.2
+urllib3==1.22
```
Comment (varunarora) [inline portal/requirements.txt]: Is this list a freeze or did you manually build it up?

**Comment by jetfuel** [comment]
LGTM

File: INSTALL.md (hunk 1 above)
Comment (cs2be) [reply INSTALL.md]: already did this

**Pull request merged**